versioned/store-spi/src/main/java/com/dremio/nessie/versioned/impl/condition/ExpressionFunction.java

```diff
@@ -63,7 +63,7 @@
   private final FunctionName name;
   private final List<Value> arguments;
 
-  private ExpressionFunction(FunctionName name, ImmutableList<Value> arguments) {
+  ExpressionFunction(FunctionName name, ImmutableList<Value> arguments) {
```

Comment:
Please add @VisibleForTesting if you're making this change for that reason.

Reply:
Done.